packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -9,52 +8,29 @@ import type { BaseButtonProps } from './BaseButton';
 export type ButtonMinHeight = 28 | 32 | 36 | 48;
 
 export type ButtonTypography = {
-  [Key in TypographyPlatforms]: {
-    fonts: {
-      size: Record<
-        NonNullable<BaseButtonProps['size']>,
-        keyof ThemeTokens['typography'][Key]['fonts']['size']
-      >;
-    };
-    lineHeights: Record<
-      NonNullable<BaseButtonProps['size']>,
-      keyof ThemeTokens['typography'][Key]['lineHeights']
-    >;
+  fonts: {
+    size: Record<NonNullable<BaseButtonProps['size']>, keyof Theme['typography']['fonts']['size']>;
   };
+  lineHeights: Record<
+    NonNullable<BaseButtonProps['size']>,
+    keyof Theme['typography']['lineHeights']
+  >;
 };
 
 const typography: ButtonTypography = {
-  onDesktop: {
-    fonts: {
-      size: {
-        xsmall: 75,
-        small: 75,
-        medium: 100,
-        large: 200,
-      },
-    },
-    lineHeights: {
-      xsmall: 'l',
-      small: 'l',
-      medium: 'l',
-      large: 's',
+  fonts: {
+    size: {
+      xsmall: 75,
+      small: 75,
+      medium: 100,
+      large: 200,
     },
   },
-  onMobile: {
-    fonts: {
-      size: {
-        xsmall: 50,
-        small: 50,
-        medium: 100,
-        large: 200,
-      },
-    },
-    lineHeights: {
-      xsmall: 's',
-      small: 's',
-      medium: 'm',
-      large: 'm',
-    },
+  lineHeights: {
+    xsmall: 's',
+    small: 's',
+    medium: 'l',
+    large: 'm',
```

Comment:
Didn't quite understand the rationale for making medium `l` and large `m`

Reply:
the left hand side are sizes of the button, not to be confused with the exact literal :D

Reply:
Does it imply having a large line height `l` for a `medium` button and so on?

Reply:
valid. @saurav12 can you check this once?

Reply:
thanks for pointing this out.
Discussed with Saurav and we'll map

button - medium lineheight to `m`
button - large lineheight to `l`